include/posets/concepts.hh

```diff
@@ -26,10 +26,10 @@ namespace posets {
   template <typename T, typename V = typename T::value_type>
   concept Downset = std::ranges::range<T> and
     not std::is_default_constructible_v<T> and
-    //std::is_constructible_v<T, size_t> and          // give the dimension
-    //std::is_constructible_v<T, size_t, size_t> and  // and give the size guess
-    std::is_constructible_v<T, V&&> and
-    std::is_constructible_v<T, std::vector<V>&&> and
+    // NOTE: Constructors take a vector, or vector of vectors, and a bound on
+    // the infinity-norm
+    std::is_constructible_v<T, V&&, unsigned> and
+    std::is_constructible_v<T, std::vector<V>&&, unsigned> and
```

Comment:
`unsigned` is not OK.  Please only use:
- [`size_t` et al.](https://en.cppreference.com/w/cpp/types/size_t)
- [fixed sized integers](https://en.cppreference.com/w/cpp/types/integer)

Beyond that, it is not OK to require that all types have an `unsigned` argument that is not used by 90% of implementations.

Reply:
that one is my fault :D dirty fix I'll explain during our call

Reply:
Let's make it V::value_type